include/bluetooth/gatt.h

```diff
@@ -706,10 +706,18 @@ struct _bt_gatt_ccc {
 	/** Highest value of all connected peer's subscriptions */
 	uint16_t value;
 
-	/** @brief CCC attribute changed callback
+	/** @brief React to presence of subscribers
 	 *
-	 *  @param attr   The attribute that's changed value
-	 *  @param value  New value
+	 *  This event signals whether the Bluetooth stack has recipients for
+	 *  bt_gatt_indicate() or bt_gatt_notify() for @p attr with @p conn set to
+	 *  @c NULL.
```

Comment:
What `@p conn` are you referring to here?

Reply:
> What @p conn are you referring to here?

The `conn` parameter of `bt_gatt_indicate` and `bt_gatt_notify`.